index.html

```diff
@@ -81,19 +81,20 @@ <h4>Malwine <div class="location">Berlin • DE</div></h4>
           </div>
           <div class="item">
             <div class="photo">
-              <a href="https://twitter.com/filtercake">
+              <a href="https://twitter.com/afknapping">
                 <img src="static/img/fabian.jpg">
               </a>
             </div>
             <div class="profile">
-              <h4>Fabian <div class="location">Hamburg • DE</div></h4>
+              <h4>Fabian <div class="location">near Hamburg, Germany</div></h4>
```

Comment:
Hi Fabian, 
sorry for getting back so late. It would be great if we could be consistent with how we display the country. 
I suggest `near Hamburg • DE`
What do you think? :) If you have another suggestion or concerns I'll be happy to hear them.